scripts/jenkins/ardana/ansible/roles/ardana_deploy/tasks/install_lint.yml

```diff
@@ -0,0 +1,30 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- name: Install ansible-lint
+  pip:
+    name: 'ansible-lint==2.1.3'
+    virtualenv: '{{ lint_venv }}'
+    virtualenv_site_packages: true
+
+- name: Install Ardana linting rules
+  synchronize:
+    delete: true
+    src: lint-rules
+    dest: "{{ lint_venv }}/lint-rules"
+
+- set_fact:
+    lint_params: '-p -x templating,resources,ANSIBLE0002 -r {{ lint_venv }}/lint-rules -r {{ lint_venv }}/lib/python2.7/site-packages/ansiblelint/rules'
```

Comment:
Why we you use set_fact for this? Could't it be a variable on `defaults/main.yml` ?